components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -6644,6 +6644,41 @@ definitions:
       groupId:
         type: string
         example: ""
+#-----------------------------------------------------
+# The Application Info resource
+#-----------------------------------------------------
+  ApplicationInfo:
+    title: Application info object with basic application details
+    properties:
+      applicationId:
+        type: string
+        example: 01234567-0123-0123-0123-012345678901
+      name:
+        type: string
+        example: CalculatorApp
+      subscriber:
+        type: string
+        example: admin
+      throttlingPolicy:
+        type: string
+        example: Unlimited
+      description:
+        type: string
+        example: Sample calculator application
+      status:
```

Comment:
Shall we remove this property?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/6772